Content.Server/GameObjects/Components/Cargo/CargoOrderDatabaseComponent.cs

```diff
@@ -22,6 +22,11 @@ public override void Initialize()
             _cargoOrderDataManager.AddComponent(this);
         }
 
+        public int[] GetCapacity()
+        {
+            return new int[2] {Database.CurrentOrderSize,Database.MaxOrderSize};
+        }
```

Comment:
It looks like the capacity array is always the same size. I think you probably want to use a tuple type like `(int, int)` instead - it's a fixed length, and makes it clearer how many values you intend to have.

Reply:
good call, will do. yeah it was always intended that it'd be fixed size

Reply:
honestly i forgot tuples were a thing

Reply:
ok tuples are in
let me know if i did that right lmao